webservice/template/admin/index.tmpl

```diff
@@ -1,5 +1,9 @@
 {{define "title"}}Dashboard{{end}}
-{{define "head"}}{{end}}
+{{define "head"}}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
these should be in the "foot", and are in the base.tmpl!?

Reply:
If I move both or either onw of theese to `foot`, the tooltip won't work properly

Reply:
This should not be done, cause you force the client to load these libs 2x times 👎